include/boost/beast/_experimental/core/ssl_stream.hpp

```diff
@@ -125,6 +131,9 @@ class ssl_stream
         return *this;
     }
 
+    /// Default Deconstructor
+    ~ssl_stream() = default;
```

Comment:
hmm...I'm not sure that this actually changes anything, the copy members were already implicitly disabled because they are not available in the underlying stream. And the destructor was already implicitly defaulted. What was the intent here?

Reply:
To be more explicit, but you're right, it's redundant since they are not available in the underlying stream.

Reply:
The less code the better, especially because defaulted special members can sometimes confuse the coverage reports.

Reply:
So do you prefer having the move ctor and move assignment to not use the '= default'?

Reply:
I think it is better to just leave them unspecified and let the compiler implicitly delete them (or generate them, whatever the case may be). The rules for what the compiler does in each case are in this presentation from @HowardHinnant https://stackoverflow.com/questions/24342941/what-are-the-rules-for-automatic-generation-of-move-operations

Reply:
That diagram is helpful, thanks.

Reply:
I tend to forget so don't be surprised if you find several places in beast where I have unnecessary explicitly defaulted special members.